mscore/palette.cpp

```diff
@@ -107,8 +107,8 @@ Palette::Palette(std::unique_ptr<PalettePanel> pp, QWidget* parent)
       setDrawGrid(pp->drawGrid());
       setMoreElements(pp->moreElements());
 
-      const auto cells = pp->takeCells(0, pp->ncells());
-      for (const PaletteCellPtr& cell : cells) {
+      const auto cs = pp->takeCells(0, pp->ncells());
```

Comment:
I would call it `allPaletteCells` or `allCells`.

Reply:
OK